FeedStoreChallenge/CoreDataFeedStore.swift

```diff
@@ -28,15 +28,49 @@ public final class CoreDataFeedStore: FeedStore {
 		context = container.newBackgroundContext()
 	}
 
-	public func retrieve(completion: @escaping RetrievalCompletion) {
-		fatalError("Must be implemented")
+	func retrieve(completion: @escaping RetrievalCompletion) {
```

Comment:
This method should be `public` to be visible from outside of the module. Otherwise it would not be available for usage by module's clients.

Please make it `public`.

```suggestion
	public func retrieve(completion: @escaping RetrievalCompletion) {
```